.teamcity/_self/projects/WebApp.kt

```diff
@@ -1,8 +1,171 @@
 package _self.projects
 
+import _self.bashNodeScript
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildStep
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
 import jetbrains.buildServer.configs.kotlin.v2019_2.Project
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.PullRequests
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.commitStatusPublisher
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.perfmon
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.pullRequests
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.vcs
 
 object WebApp : Project({
 	id("WebApp")
 	name = "Web app"
+
+	buildType(RunCalypsoE2eDesktopTests)
+})
+
+object RunCalypsoE2eDesktopTests : BuildType({
+	uuid = "52f38738-92b2-43cb-b7fb-19fce03cb67c"
```

Comment:
I think this `uuid` is what makes sure TeamCity understands this is moving an existing build and not creating a new one. Generated via exporting the project in TeamCity UI.

Reply:
From the docs:

> uuid
> BuildType or template uuid. TeamCity uses it to identify the entity. If uuid changes, TeamCity considers it to be a new entity. If uuid is missing, it is reconstructed from entity id. Uuid must be unique across all entities on the server.

> id
> BuildType or template id. It appears in the web UI and is used in urls. If the entity has a uuid specified, then the id can be changed at any time. If uuid is omitted, then TeamCity treats an entity with a changed id as a new entity, all data associated with the old entity will be lost (e.g. a build history). Id can also be used by some settings, e.g. as a part of parameter reference. If you change the id, you should find all its occurrences in the current project and change them too. Id must be unique across all buildTypes and templates on the server. If id is missing, it will be generated from the class name (if the class is not from the jetbrains.buildServer.configs.kotlin.v2018_1 package).